pages/templates/pages/microsite_page.html

```diff
@@ -1,12 +1,210 @@
+      {% endif %}
+
+      {% if page.primary_content_background_image %}
+        {% image page.primary_content_background_image original as primary_img %}
+        background-image: url({{ primary_img.url }});
+      {% endif %}
+    }
+
+    #primary_content a {
+      {% if page.primary_content_text_color %}
+        color: #{{ page.primary_content_text_color }};
+      {% endif %}
+    }
+
+    #secondary_content {
+      background-size: cover;
+      {% if page.secondary_content_background_color %}
+        background-color: #{{ page.secondary_content_background_color }};
+      {% endif %}
+
+      {% if page.secondary_content_text_color %}
+        color: #{{ page.secondary_content_text_color }};
+      {% endif %}
+
+      {% if page.secondary_content_background_image %}
+        {% image page.secondary_content_background_image original as secondary_img %}
+        background-image: url({{ secondary_img.url }});
+      {% endif %}
+    }
+
+    #secondary_content a {
+      {% if page.secondary_content_text_color %}
+        color: #{{ page.secondary_content_text_color }};
+      {% endif %}
+    }
+
+    #footer, .microsite .footer {
+      background-size: cover;
+      {% if page.custom_footer_background_color %}
+        background-color: #{{ page.custom_footer_background_color }};
+      {% endif %}
+
+      {% if page.custom_footer_text_color %}
+        color: #{{ page.custom_footer_text_color }};
+      {% endif %}
+
+      {% if page.custom_footer_background_image %}
+        {% image page.custom_footer_background_image original as secondary_img %}
+        background-image: url({{ secondary_img.url }});
+      {% endif %}
+    }
+
+    #footer a, .microsite .footer a {
+      {% if page.custom_footer_text_color %}
+        color: #{{ page.custom_footer_text_color }};
+      {% endif %}
+    }
+
+    .microsite-content .can_button, .microsite-content #can_embed_form input[type="submit"], .microsite-content #can_embed_form .button, .microsite-content #donate_auto_modal input[type="submit"], .microsite-content #donate_auto_modal .button {
+      {% if page.button_background_color %}
+        background-color: #{{page.button_background_color}} !important;
+      {% endif %}
+
+      {% if page.button_text_color %}
+        color: #{{page.button_text_color}} !important;
+      {% endif %}
+    }
+  </style>
+{% endblock %}
+
+<div class="microsite">
+
+{% block notification_banner %}
+<!-- TODO: conditional notification banner on page model -->
+{% endblock %}
+
+{% block nav %}
+  {% if page.standard_header_show %}
+    {{ block.super }}
+  {% endif %}
+{% endblock %}
 
 {% block content %}
-{% if page.secondary_content_show %}
-<div style="background-color: #{% firstof page.secondary_content_background_color 'FFFFFF' %}">
-	{{ page.secondary_content | richtext }}
-</div>
-{% endif %}
+  <div id="primary_content" class="container-fluid microsite-content">
+    {% if page.custom_header_show %}
```

Comment:
Custom header is not part of primary content in the model, so this seems confusing?

Reply:
This is so you can have a transparent nav with primary background image model behind it if one should choose.

Reply:
I thought the idea was to have the primary background image just for primary content area, not the nav/header area?

Reply:
The nav should be a part of primary content area so we can enable layouts like these
![2017-11-19-21-54-localhost_8000](https://user-images.githubusercontent.com/3048335/33041850-6051139c-ce05-11e7-8ba4-494b8770c514.png).

Reply:
I think idea is fine but not what I had in mind. We may want to update the model.